docs/userguide_spark2.1.md

```diff
@@ -277,10 +242,35 @@ val tisparkDF = spark.sql("select * from tispark_table").toDF
 tisparkDF.write.saveAsTable("hive_table") // save table to hive
 spark.sql("select * from hive_table a, tispark_table b where a.col1 = b.col1").show // join table across Hive and Tispark
 ```
+## Load Spark DataFrame into TiDB using TiDB Connector
+TiSpark natively supports writing data to TiKV using Spark Data Source API. ACID is also guaranteed.
```

Comment:
```suggestion
TiSpark natively supports writing data to TiKV via Spark Data Source API and guarantees ACID.
```
or
```suggestion
TiSpark natively supports writing data to TiKV via Spark Data Source API and ACID guarantees.
```

Reply:
2.1 does not support native write right? Why here doc for 2.1 has this?

Reply:
fixed